libs/domains/environment/src/lib/slices/environment-deployment-rules.slice.spec.ts

```diff
@@ -0,0 +1,51 @@
+import {
+  environmentDeploymentRulesAdapter,
+  environmentDeploymentRulesReducer,
+} from './environment-deployment-rules.slice'
+
+describe('environmentDeploymentRules reducer', () => {
+  it('should handle initial state', () => {
+    const expected = environmentDeploymentRulesAdapter.getInitialState({
+      loadingStatus: 'not loaded',
+      joinEnvironmentDeploymentRules: {},
+      error: null,
+    })
+
+    expect(environmentDeploymentRulesReducer(undefined, { type: '' })).toEqual(expected)
+  })
+
+  // it('should handle fetchEnvironmentDeploymentRules', () => {
```

Comment:
What do we do with these comments? Either we decide that we don't want to test that, or then we have an example of an almost fully tested store with environment-variable